finagle-mysql/src/main/scala/com/twitter/finagle/mysql/Request.scala

```diff
@@ -151,33 +151,31 @@ case class HandshakeResponse(
  * a prepared statement.
  * [[http://dev.mysql.com/doc/internals/en/com-stmt-execute.html]]
  */
-case class ExecuteRequest(
-  stmtId: Int,
-  params: IndexedSeq[Any] = IndexedSeq.empty,
-  hasNewParams: Boolean = true,
-  flags: Byte = 0
+class ExecuteRequest(
```

Comment:
@mosesn Because of the sanitization that's needed beforehand. With a regular class, I could mark the constructor private. With a case class, I still could do that, but the `apply` auto-generated in the companion object will be publicly available, and it cannot even be overridden.